Mathlib/Tactic/SolveByElim.lean

```diff
@@ -76,11 +76,28 @@ do
   let locals : TermElabM (List Expr) := if noDflt then pure [] else pure (← getLocalHyps).toList
   return (hs, locals)
 
+def mkAssumptionSet' (_noDflt : Bool) (hs : List (TSyntax `term)) :
```

Comment:
It looks to me like this is a copy of `mkAssumptionSet` but with `noDflt` hardcoded to `true`. If that's the case, then why is it needed?
Please delete or add a doc comment explaining.

Reply:
Sorry, I was thinking I had a good version, but then Jireh's use-case did not work, so I went on and experimented with some things, but I did find out how to fix it.
So for now you can safely ignore my request for review.